pages/star-search/index.tsx

```diff
@@ -79,7 +77,7 @@ export default function StarSearchPage({ userId, bearerToken, ogImageUrl }: Star
   const [chat, setChat] = useState<StarSearchChat[]>([]);
   const inputRef = useRef<HTMLInputElement>(null);
   const [isRunning, setIsRunning] = useState(false);
-  const isMobile = useMediaQuery("(max-width: 576px)");
+  const isMobile = useMediaQuery("(max-width: 768px)");
```

Comment:
I increased the size here to the medium breakpoint because the current scrollable suggestions were outside of the viewport on size larger than the small breakpoint, but smaller than the medium breakpoint.